custom/icds_reports/tests/agg_tests/reports/test_prevalence_of_severe.py

```diff
@@ -727,498 +727,3 @@ def test_sector_data_chart_data(self):
                 }
             ]
         )
-
-
-@override_settings(SERVER_ENVIRONMENT='icds')
-class TestPrevalenceOfSevereICDSFeatureFlag(TestCase):
```

Comment:
can you say why this was a cleanup? Whats the test case this was testing and why isn't it needed anymore?

Reply:
hey @emord, All test cases in this class are just replication of all test cases in the class above just calling the function with feature flag on. and those functions are no more under the feature flag which makes these two sets of classes the same as each other in terms of the test they do.